### PR TITLE
Update location api endpoint

### PR DESCRIPTION
**Pull request by iloveicedgreentea**

### CODE REVIEW

File: application/database/database_routes.py [hunk 1]
```diff
@@ -281,7 +281,6 @@ def update_location(location_id):
   """
```
Comment (jorge-caballero) [inline application/database/database_routes.py:294]: @iloveicedgreentea i may be missing it, but the code does not update the `updated_on` field when a `PUT` request succeeds

**Review by jorge-caballero — CHANGES_REQUESTED** [review]
- [ ] update the `updated_on` field when a `PUT` request succeeds

**Comment by iloveicedgreentea** [comment]
Fixed some minor things, added automatic migrations

**Review by jorge-caballero — APPROVED** [review]
LGTM, but honestly not the best to review migrations written in python...it's been little while since I've been in that mode.

**Review by jorge-caballero — APPROVED** [review]
LGTM.

**Pull request merged**